src/api/entitlements.ts

```diff
@@ -231,9 +231,11 @@ export class Entitlements {
    * When a metering entitlement is consumed, SwG shows the user a metering dialog.
    * When the user closes the dialog, SwG depletes one of the user's remaining
    * "free reads".
-   * @param {?Function=} onCloseDialog Called after the user closes the dialog.
    */
-  consume(onCloseDialog?: Function | null) {
+  consume(
+    /** Called after the user closes the dialog. */
```

Comment:
This comment should be a param in the JSDoc.

Reply:
Updated